sdks/java/io/hadoop/jdk1.8-tests/src/test/resources/kubernetes/cassandra/LargeProductionCluster/start-up.sh

```diff
@@ -0,0 +1,24 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/bin/sh
+
+# Create Cassandra services and Replication controller.
+kubectl create -f cassandra-svc-rc.yaml
+# scale cassandra cluster upto 3 nodes
+kubectl scale rc cassandra --replicas=3
+# Wait until all replica pods are up and running.
+sleep 3m
```

Comment:
We will work on this. But as per discussion in call, now we are supposed to remove Largecluster from cassandra for this PR.